tests/unit/client_test.py

```diff
@@ -1,15 +1,143 @@
 import os
+
+import sure
```

Comment:
This is the import that magically gives you the [BDD-style assertions](http://falcao.it/sure/). You might hate them, in which case I'm happy swapping them out with more standard assertions. I find them very readable.